conda/recipes/libcucim/meta.yaml

```
@@ -69,8 +70,10 @@ requirements:
     - libcufile
     - cuda-cudart
     - libnvjpeg
+    - libnvimgcodec0 {{ nvimgcodec_version }}  # nvImageCodec runtime library
   run_constrained:
     - {{ pin_compatible('openslide') }}
+    - libnvimgcodec-dev {{ nvimgcodec_version }}  # Optional: for development/debugging
```

```suggestion
    - libnvjpeg
  run_constrained:
    - {{ pin_compatible('openslide') }}
```
These constraints aren't necessary. `libnvimgcodec-dev` has appropriate run_exports, and your constraints are less strict than the run_exports.